src/sections/General/Navigation/navigation.style.js

```diff
@@ -426,7 +426,17 @@ const NavigationWrap = styled.header`
       padding-left: 10px;
     }
   }
+
+  .nav-link-active-mobile {
+    color: ${(props) => props.theme.menuHoverColor};
+  }
   
+  .nav-link-active {
+    color: ${(props) => props.theme.menuHoverColor};
+    border-bottom: 1px solid ${(props) => props.theme.menuHoverColor};
```

Comment:
Let's avoid the bottom border for active links, and just have it highlighted (i.e, change the color). This would mean that ultimately we can get rid of 2 separate classes for mobile, desktop views.